sufia-models/app/services/sufia/generic_file_indexing_service.rb

```diff
@@ -6,6 +6,7 @@ def generate_solr_document
         solr_doc[Solrizer.solr_name('file_format')] = object.file_format
         solr_doc[Solrizer.solr_name('file_format', :facetable)] = object.file_format
         solr_doc['all_text_timv'] = object.full_text.content
+        solr_doc[Solrizer.solr_name('file_size', Solrizer::Descriptor.new(:integer, :stored);)] = object.content.size.to_i
```

Comment:
I think a stray semi-colon may have crept in between the two closing parens here.